.github/workflows/fmt-and-lint.yml

```diff
@@ -18,12 +18,13 @@ jobs:
       run: |
         rustup toolchain install nightly-2022-10-29 --no-self-update --profile=minimal --component rustfmt clippy
         rustup default nightly-2022-10-29
+        apt-get update && apt-get -y install libudev-dev
```

Comment:
This requires root permission, and you have to run `update` on non-promt mode as well. e.g `sudo apt-get update -y`

With the note above, can you also remove this from toolchain section and create new one like:

```yml
- name: Install OS dependencies
  run: |
    sudo apt-get update -y
    sudo apt-get install -y libudev-dev
```

Reply:
done